packages/ui/components/input-tel-dropdown/test-suites/LionInputTelDropdown.suite.js

```diff
@@ -335,12 +335,14 @@ export function runInputTelDropdownSuite({ klass } = { klass: LionInputTelDropdo
         expect(el.value).to.equal('+32');
       });
 
-      it('focuses the textbox right after selection if selected via opened dropdown', async () => {
+      it('focuses the textbox right after selection if selected via opened dropdown if interaction-mode is mac', async () => {
         const el = await fixture(
-          html` <${tag} .allowedRegions="${[
-            'NL',
-            'BE',
-          ]}" .modelValue="${'+31612345678'}"></${tag}> `,
+          html`
+            <${tag}
```

Comment:
Shouldn't we set `interaction-mode` in this fixture?

Reply:
Edit: I think we do that in `WithFormControlInputTelDropdown`

Reply:
Maybe we can do it here explicitly and then also do the windows/linux test for clarity?

Reply:
Just to get it clear, you want to recreate 2 `WithFormControlInputTelDropdown` elements in this test suite 1 for `mac` and 1 for `windows/linux`?

Reply:
Yeah indeed  (if changing on the fly doesn't work)